extension/leaderelector/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+type: k8s_leader_elector
```

Comment:
Please update the module name as well `leaderelector` -> `k8sleaderelector`

Reply:
done